commands/store.js

```diff
@@ -53,7 +53,7 @@ class Store extends Command {
             }
             await this.client.db.giveUserItem(connection, message.guild.id, message.author.id, storeItem.id, amount);
             await connection.query('COMMIT');
-            return message.channel.send(`${message.author} You have bought ${amount}x ${storeItem.name} :tada:`);
+            return message.channel.send(`${message.author} You have bought ${amount}x ${storeItem.name} :tada:\nYou now have <:blobcoin:386630453224013824> ${userData.currency}.`);
```

Comment:
userData is not in this scope, but if it was, it would be out-of-date anyway.

Reply:
In this particular instance, `deducted` already contains the result from `takeUserCurrency`, which returns an anon object including the `currency` field for the user. You should use `${deducted.currency}` instead.